uiowa/uiowa
# Headline alignment options for blocks

Closes #4402 
Closes #2427

# Story

As a site owner, I want to align block headlines to the center as well as to the left. 
Pre-existing blocks should have Left (default) set as their alignment.

# How to test


- Test a fresh site install and make sure default content can be modified and saved.
- Add new blocks and make sure switching between alignments works fine.
- Also check that switching alignments works on pre-existing blocks.
- `blt ds --site=grad.uiowa.edu` to test the Academic Dates block
- `blt ds --site=recserv.uiowa.edu` to test the Hours block.
- Blocks with the alignment option:
  - Collection 
  - Events
  - Featured Content
  - Image Gallery
  - RSS Feeds
  - Slider
  - Timeline
  - Text Area
  - Articles Listblock
  - People Listblock
  - Academic Dates (MAUI)
  - Hours
  - Sign Up Form (Sitenow Dispatch)
  - Thank You Form (Sitenow Dispatch)
